pallets/msa/src/types.rs

```diff
@@ -16,3 +16,23 @@ pub struct KeyInfo<BlockNumber> {
 	pub nonce: u32,
 	pub expired: BlockNumber,
 }
+
+impl<BlockNumber> KeyInfo<BlockNumber>
```

Comment:
for future thoughts maybe we have a trait and implement that trait on this type.

Reply:
yeah, that would be nice. Will refactor in future tickets.